source/Pendulum_en.rst

```diff
@@ -0,0 +1,186 @@
+Pendulum

```

Comment:
all the file should use lower case and underscore nomenclature, like `pendulum_en.rst`